.gitignore

```
@@ -1,16 +1,16 @@
-/target/
+target/
```

```suggestion
/target/
/*/target/
```
It is better to be explicit about which `target` folders you want to ignore. A folder called "target" might conceivably be created on purposes somewhere in the source tree and it might then not immediately be obvious why it was left out of git.

---

In that case shouldn't the first rule `/target/` be left out as well? I believe the current maven configuration will not generate this folder anymore.

---

It might be left out, but the pattern doesn't do any harm either, as its scope is limited. *If* a target folder were generated in the top level project directory it *would* need to be ignored.